currency-exchange-service/src/main/java/com/ubaid/ms/currencyexchangeservice/controller/CurrencyExchangeController.java

```diff
@@ -5,14 +5,16 @@
 import com.ubaid.ms.currencyexchangeservice.service.ExchangeRateService;
 import io.swagger.annotations.*;
 import lombok.RequiredArgsConstructor;
-import org.springframework.http.MediaType;
 import org.springframework.security.access.prepost.PreAuthorize;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
-import java.net.HttpURLConnection;
+import static com.ubaid.ms.common.Constants.BEARER;
+import static java.net.HttpURLConnection.HTTP_OK;
+import static java.net.HttpURLConnection.HTTP_UNAUTHORIZED;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
```

Comment:
move it to constant file and import from constant file instead of directly importing